### PR TITLE
fix: Import Export course content

### PR DESCRIPTION
**Pull request by CodeWithEmad**
I was reading this section and noticed some issues.

- images are updated
- content changed a bit for better readability
- 1 outdated/redirected link fixed.

### CODE REVIEW

File: source/educators/migration_wip/images/course_export_page.png
Comment (jswope00) [inline source/educators/migration_wip/images/course_export_page.png:1]: The deleted image in this case is the new MFE, whereas the new image shows the old non-MFE front-end. We want the docs to use the latest front-ends, so this change should be reverted.

File: source/educators/migration_wip/images/course_import_page.png
Comment (jswope00) [inline source/educators/migration_wip/images/course_import_page.png:1]: The deleted image in this case is the new MFE, whereas the new image shows the old non-MFE front-end. We want the docs to use the latest front-ends, so this change should be reverted.

File: source/educators/migration_wip/15_releasing_course/export_import_course.rst [hunk 1]
```diff
@@ -76,7 +76,7 @@ Course Outline Terminology in Exported Files
 
 Sections, subsections, units, and components have different names in the Studio
 **Course Outline** view and in the list of files that you see after you
-export your course and open the .xml files for editing. The following table
+export your course and open the **.xml** files for editing. The following table
```
[the remark this answers is not in the document]
Comment (jswope00) [reply source/educators/migration_wip/15_releasing_course/export_import_course.rst:79]: I think yes, this should also be double backticked.

File: source/educators/migration_wip/images/course_export_page.png
Comment (jswope00) [reply source/educators/migration_wip/images/course_export_page.png:1]: See this comment.

**Review by jswope00 — CHANGES_REQUESTED** [review]
Thanks for your suggestions! I have a few comments inline that need to be addressed and then I think this is good.

File: source/educators/migration_wip/15_releasing_course/export_import_course.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (CodeWithEmad) [reply source/educators/migration_wip/15_releasing_course/export_import_course.rst]: fixed

**Review by jswope00 — APPROVED** [review]
Approved!

**Comment by openedx-webhooks** [comment]
@CodeWithEmad 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/docs.openedx.org&entry.867055334=https://github.com/openedx/docs.openedx.org/pull/502&entry.1484655318=https://github.com/CodeWithEmad&entry.752974735=2024-06-05+08:42&entry.1917517419=2024-06-11+14:47&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**